src/integrationTest/java/uk/gov/hmcts/reform/roleassignment/config/SwaggerPublisher.java

```diff
@@ -20,27 +20,31 @@
  * Built-in feature which saves service's swagger specs in temporary directory.
  * Each travis run on master should automatically save and upload (if updated) documentation.
  */
-@SpringJUnitWebConfig
-@SpringBootTest
-@AutoConfigureMockMvc
-class SwaggerPublisher {
+public class SwaggerPublisher extends BaseTest {
+
+    private MockMvc mockMvc;
 
-    @Autowired
-    private transient MockMvc mvc;
+    @Inject
+    private WebApplicationContext wac;
+
+    @Before
+    public void setUp() throws Exception {
+        mockMvc = MockMvcBuilders.webAppContextSetup(wac).build();
+        MockitoAnnotations.openMocks(this);
+    }
 
-    @DisplayName("Generate swagger documentation")
     @Test
     @Disabled
```

Comment:
Should it realy be disabled?

Reply:
@Disabled removed

Reply:
Resolved in e98225bd1c69790afa414a0ddf58d3d65cb3cead